contribute/README.md

```diff
@@ -10,9 +10,11 @@ Thanks for enhancing OpenFisca anyway!
 
 ## Why contribute to OpenFisca?
 
-OpenFisca is a project being developed under the GPLv3 license or later.
+OpenFisca is a project being developed under the AGPL-3.0 license.
 The source code is freely available and modifiable.
 
+We encourage improvements and evolutions of the software to be shared back, for it allows the community to grow through improvements to the core engine and updates to the modeled tax-benefix systems.
```

Comment:
Don't they *have to* share any change to the software ?

```
The GNU Affero General Public License is designed specifically to ensure that, in such cases, the modified source code becomes available to the community. It requires the operator of a network server to provide the source code of the modified version running there to the users of that server. Therefore, public use of a modified version, on a publicly accessible server, gives the public access to the source code of the modified version.
```
https://www.gnu.org/licenses/agpl-3.0.fr.html
https://softwareengineering.stackexchange.com/questions/107883/agpl-what-you-can-do-and-what-you-cant

Reply:
I think it is not contradictory with the verbatim, although I agree it is not detailed enough about the _strongness_ of the encouragement.